res/layout/conversation_list_item_view.xml

```diff
@@ -88,10 +88,24 @@
             <LinearLayout
                 android:orientation="horizontal"
                 android:minHeight="21dp"
+                android:showDividers="middle"
+                android:divider="@drawable/badge_divider"
```

Comment:
this is an elegant solution! i was not aware that `android:divider` this exist :)

i just double checked https://developer.android.com/reference/android/widget/LinearLayout - dividers were revamped in API 11, however, seems to exist forever - so, in any case, no issue with out [API 16](https://github.com/deltachat/deltachat-android/pull/1227) :)